app/src/common/shared/org/mozilla/vrbrowser/telemetry/metrics/FxA.java

```diff
@@ -0,0 +1,41 @@
+package org.mozilla.vrbrowser.telemetry.metrics;
+
+import androidx.annotation.NonNull;
+
+import org.mozilla.vrbrowser.GleanMetrics.Fxa;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class FxA {
```

Comment:
I would like to suggest to merge these functions into `GleanMetricsService.java`.

Reply:
I'd rather having different namespaces for each category so we don't end up with a big `GleanMetricsService` class with a lot of long method names just to avoid collisions or confusion.

Glean actually does the same internally when generating the Metrics Kotlin classes.